db/migrate/20170330161442_change_institution_defaults.rb

```diff
@@ -0,0 +1,48 @@
+class ChangeInstitutionDefaults < ActiveRecord::Migration
+  def change
+    change_table :institutions do |t|
+      t.change :flight, :boolean, default: nil
+      t.change :correspondence, :boolean, default: nil
+      t.change :gibill, :integer, default: nil
+      t.change :yr, :boolean, default: nil
```

Comment:
This presents an issue for the not `caution_flag` search. I didn't realize that was added to the functionality, did someone request that? In this case there would be an issue.